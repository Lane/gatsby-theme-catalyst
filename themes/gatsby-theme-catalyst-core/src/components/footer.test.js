import React from "react"
import renderer from "react-test-renderer"
import SiteFooter from "./footer"
describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SiteFooter />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})