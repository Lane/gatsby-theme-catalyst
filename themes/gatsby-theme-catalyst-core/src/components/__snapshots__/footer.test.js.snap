// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="css-o72uzv-SiteFooter"
  id="footer"
>
  <p>
    Footer area for latent shadowing in sibling themes
  </p>
</footer>
`;
